docs/src/install/core.rst

```diff
@@ -37,9 +43,8 @@ also recommended, but not necessary. Example:
 .. code-block:: sh
 
    <Create and navigate to the directory where local repo will reside>
-   git clone https://github.com/mila-udem/orion.git
+   git clone https://github.com/mila-udem/orion.git --branch develop
    mkvirtualenv -a $PWD/orion orion
-   workon orion
```

Comment:
Why is the removed?

Reply:
Because mkvirtualenv automatically sources the virtualenv, you don't need workon right after mkvirtualenv.